forge-gui/res/cardsfolder/upcoming/pact_weapon.txt

```diff
@@ -0,0 +1,16 @@
+Name:Pact Weapon
+ManaCost:3 B
+Types:Artifact Equipment
+K:Equip:Discard<1/Card>
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | IsPresent$ Card.Self+AttachedTo Creature | AddKeyword$ You don't lose the game for having 0 or less life. | Description$ As long as CARDNAME is attached to a creature, you don't lose the game for having 0 or less life.
+T:Mode$ Attacks | ValidCard$ Card.EquippedBy | Execute$ TrigDraw | TriggerDescription$ Whenever equipped creature attacks, draw a card and reveal it. The creature gets +X/+X until end of turn and you lose X life, where X is that card's mana value.
+SVar:TrigDraw:DB$ Draw | NumCards$ 1 | Reveal$ True | RememberDrawn$ True | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | ValidTgts$ Creature.EquippedBy | NumAtt$ +X | NumDef$ +X | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ X | Defined$ You | SubAbility$ DBClean
+SVar:DBClean:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Remembered$CardManaCost
+SVar:AIPreference:DiscardCost$Card
+AI:RemoveDeck:Random
```

Comment:
Why this tag?

Reply:
It was on a few of the cards that I used as the basis for scripting this card, so it seemed to be a parameter I should include.

Reply:
Which ones – I don't really see a reason for it here